Automattic/vip-go-mu-plugins
# Bump Jetpack to hotfix version

Includes https://github.com/Automattic/jetpack/pull/11520 and https://github.com/Automattic/jetpack/pull/11503

## Description
Jetpack 7.1.1 added additional caching when the WordPress.com stats API did not return an expected response; however, the output changed. This resulted in a regression where the Jetpack wp-admin dashboard would fail.

This retains the caching, but returning the error, as it originally did, so the dashboard can process and return an error message.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR on a connected Jetpack site
1. Set public-api to a WP.com sandbox and cause a 500 response from the sandbox.
1. Go to `wp-admin` > `Jetpack`
1. Ensure an error message or loading indicator is displayed for the stats graph. Not a JS whitescreen fatal.
